From 49c1562e8d9ecba4239405abb1426bdfe11b82f7 Mon Sep 17 00:00:00 2001
From: Benjamin Kaduk <bkaduk@akamai.com>
Date: Tue, 7 Sep 2021 09:17:39 -0700
Subject: [PATCH 50/63] QUIC: update SSL_provide_quic_data() documentation

We now let you call this function outside of the handshake, to provide
post-handshake QUIC data.

We also no longer have the limitation that the application must provide
the TLS handshake message header in a single call.
---
 doc/man3/SSL_CTX_set_quic_method.pod | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/doc/man3/SSL_CTX_set_quic_method.pod b/doc/man3/SSL_CTX_set_quic_method.pod
index b9191a0264..b2d78872ca 100644
--- a/doc/man3/SSL_CTX_set_quic_method.pod
+++ b/doc/man3/SSL_CTX_set_quic_method.pod
@@ -82,10 +82,8 @@ SSL_quic_write_level() returns the current write encryption level.
 
 SSL_provide_quic_data() is used to provide data from QUIC CRYPTO frames to the
 state machine, at a particular encryption level B<level>. It is an error to
-call this function outside of the handshake or with an encryption level other
-than the current read level. The application must buffer and consolidate any
-frames with less than four bytes of content.  It returns one on success and
-zero on error.
+call this function with an encryption level less than the current read level.
+It returns one on success and zero on error.
 
 SSL_process_quic_post_handshake() processes any data that QUIC has provided
 after the handshake has completed. This includes NewSessionTicket messages
-- 
2.42.0

